docs/sources/network/asserts.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Set up Beyla network metrics in Kubernetes with Helm for Asserts
+menuTitle: Set up Asserts network
+description: A guide to install Beyla network metrics in Kubernetes with Helm for Asserts.
+weight: 1
+keywords:
+  - Beyla
+  - eBPF
+  - Network
+---
+
+# Set up Beyla network metrics in Kubernetes with Helm for Asserts
+
+[Asserts](/docs/grafana-cloud/monitor-applications/asserts/) works with Beyla and requires Beyla network metrics. Learn how to set up Beyla network metrics in Kubernetes with Helm to export telemetry data to Asserts.
+
+To learn more about Beyla network metrics, consult the [Network](/docs/beyla/latest/network/) documentation.
+
+## Prerequisites
+
+Before you install Beyla network metrics and export telemetry data to Asserts you need:
+
+1. A free Grafana Cloud account.
+1. An application to auto-instrument with Beyla.
+1. A Linux environment that supports eBPF kernel modules.
+1. Administrative `sudo` privileges or `CAP_SYS_ADMIN` permissions.
```

Comment:
This sentence is not needed, as the container is run in _privileged_ mode but the user that runs kubernetes does not need any special permission, in terms of operating system access.

I'd remove it or, maybe being too obvious, rewriting it to:
```suggestion
1. Access rights to a Kubernetes cluster, enough to create components with privileges.
```